htdocs/install/mysql/migration/6.0.0-7.0.0.sql

```diff
@@ -528,6 +528,9 @@ CREATE TABLE llx_comment (
 
 DELETE FROM llx_const where name = __ENCRYPT('MAIN_SHOW_WORKBOARD')__;
 
+-- Adherent - Update old constants
+UPDATE llx_const SET value = REPLACE(value, '%', '__') WHERE name LIKE 'ADHERENT%'; 
+
```

Comment:
@atm-quentin @eldy 
I'm thinking about compatibility with the tag __ENCRYPT()__

Reply:
No idea on how we can process this for crypted columns.
If data is crypted, we must uncrypt then recrypt. But process to uncrypt is in php i think and migration is using sql. Hummm...
Making the migration in php may be a solution. Never mind, on crypted cloumns, this request will just make nothing, so it should not be annoying to do it